lib/compute/jenkins-main-node.ts

```diff
@@ -268,7 +268,7 @@ export class JenkinsMainNode {
       // Jenkins CVE https://www.jenkins.io/security/advisory/2024-01-24/ mitigation 
       InitCommand.shellCommand('mkdir -p /var/lib/jenkins/init.groovy.d'),
       // eslint-disable-next-line max-len
-      InitCommand.shellCommand('sudo wget -P /var/lib/jenkins/init.groovy.d https://raw.githubusercontent.com/jenkinsci-cert/SECURITY-3314-3315/main/disable-cli.groovy'),
+      InitCommand.shellCommand('sudo curl -SL https://raw.githubusercontent.com/jenkinsci-cert/SECURITY-3314-3315/main/disable-cli.groovy -o /var/lib/jenkins/init.groovy.d/disable-cli.groovy'),
```

Comment:
Another issue is we should use a commit to lock this file, instead of branch `main` as that would be a moving target.

Reply:
thanks ,, makes sense, and I learned that by doing this way, we increase stability and reproducibility.. thanks for the guidence!